client/src/components/AboutBanner/AboutBanner.css

```diff
@@ -0,0 +1,8 @@
+ul {
+    word-wrap: break-word;
+}
+
+img {
+    float: left;
+    size: 200px, 200px;
```

Comment:
When setting up the size of the image, you're changing the image quality if you change screen size using this format. I generally avoid doing this like this to keep the best image quality possible. I'll follow up with you in person so we can figure out how to best achieve what you wanted with this.